README.md

```diff
@@ -1 +1,3 @@
-# csi-proxy-builder
\ No newline at end of file
+# csi-proxy-builder
+
+This repository builds [csi-proxy](https://github.com/kubernetes-csi/csi-proxy) and releases binary. The binary is used for windows nodes in ansible-eks. Csi-proxy is required for [csi-driver-smb](https://github.com/kubernetes-csi/csi-proxy) on windows nodes.
```

Comment:
As I'm sure someone will notice this is public, can you make a node here, in hopes it's ready, that mentions why we even need this, and why it's public?